tests/aggregation_regress/tests.py

```diff
@@ -554,6 +554,10 @@ def test_conditional_aggregate_on_complex_condition(self):
             325,
         )
 
+    def test_q_annotation_aggregate(self):
+        # Ensure aggregation works with annotations using Q objects
```

Comment:
Docstrings should state the expected behavior and omit prefixes like "Ensure" since all tests ensure things (from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style)).

Reply:
I don't think that this comment is valuable. The test method name is self-explanatory. I'd chop it. 
```suggestion
```